Seminar 5/Seminar 5 HW/task_3.py

```diff
@@ -0,0 +1,22 @@
+# Создайте функцию генератор чисел Фибоначчи (см. Википедию)
+
+def fibonacci():
+    fib_1 = 1
+    fib_2 = 1
+    while True:
+        fib_sum = fib_1 + fib_2
+        fib_1 = fib_2
+        fib_2 = fib_sum
+        yield fib_sum
```

Comment:
Используйте парные присваивание. И кстати, почему вас 1 число ряда Фибоначчи равно единице?
```suggestion
    fib_1, fib_2  = 0, 1
    while True:
         yield fib_1
        fib_1, fib_2  = fib_2, fib_1 + fib_2

```

Reply:
Не стал использовать парные, раз уж всё равно расписываю. А про единицу, я напрочь забыл как этот Фибоначчи искать и в старых решениях с 1 было..